src/OutGridView.Gui/Services/FilterBuilder.cs

```diff
@@ -54,7 +54,7 @@ static class FilterBuilder
             //For all terms at least-one property matches
             return dataList => tokens.All(t =>
             {
-                return dataList.Values.Any(data =>
+                return dataList.Values.Select(x => x.Value).Any(data =>
```

Comment:
You could probably just get rid of the Select and just reference `data.Value`, right? Also line 60 looks like you're still using `data.Value`. Is that expected?